README.md

```diff
@@ -22,7 +22,7 @@ const result = await arcClient.getTransactionStatus(txid);
 or as commonjs module:
 
 ```javascript
-const { ArcClient } = require('js-arc-client').ArcClient;
+const { ArcClient } = require('@bitcoin-a/arc-client').ArcClient;
```

Comment:
Looks like we also need to remove one of two things here:

1. The {} braces around ArcClient on the left. OR
2. the .ArcClient at the end of this line.

Reply:
At the moment ArcClient is left undefined because it's being assigned to `exports.ArcClient.ArcClient`